### PR TITLE
Fixed kirby-button paddings when used with kirby-icon

### LINKED ISSUE #1207: [BUG] Small Button (with icon) should have 12px padding where the icon is placed
**Describe the bug**
Our small buttons have 4px too much padding where the icon is placed. 

**To Reproduce**

1. Go to https://cookbook.kirby.design/home/showcase/button
2. Inspect a small button with an icon
3. See it has 16px padding instead of 12px

**Kirby version**
v1.3.7

**Expected behavior**
Should have 12px padding

**Screenshots**
Skærmbillede 2020-11-20 kl  08 21 48

**Please complete the following information:**
All devices ;) 

**Additional context**
See also zeplin: https://zpl.io/aNXvx4Z

## Tasks:
- [x] Move issue from "Ready to do" -> "in progress" on our [project board](https://github.com/kirbydesign/designsystem/projects/1)
- [x] Create Fix Branch
- [x] Create test reproducing the error
- [x] Fix
- [x] If relevant [Update Examples and Showcase](https://cookbook.kirby.design/home/showcase/button)
- [x] Push commits to GitHub
- [x] Open a pull request (remember to add `closes #issueno` to the description) and ask @kirbydesign/kirby-guild for review
- [ ] Merge to master and delete branch

### PR DESCRIPTION
**Pull request by thupi**
## Which issue does this PR close?

This PR closes #1207

## What is the new behavior?

The kirby-button now applies that correct padding when small size kirby-button are used with kirby-icon

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/src/lib/components/button/button.component.spec.ts [hunk 1]
```diff
@@ -463,4 +463,98 @@ describe('ButtonComponent', () => {
...
+        });
+        it('should render with correct padding-inline', () => {
+          expect(element.getElementsByClassName('content-layer').length).toBe(1);
+          expect(element.getElementsByClassName('content-layer')[0]).toHaveComputedStyle({
+            'padding-inline': scenario.expected.paddingInline,
+          });
+        });
+      });
+
+      describe(`through template property binding`, () => {
```
Comment (mark-drastrup) [inline libs/designsystem/src/lib/components/button/button.component.spec.ts:553]: @thupi I have question that is probably stupid :) Isn't this `describe` block identical to the one above: `through an input property`?

**Review by mark-drastrup — COMMENTED** [review]
@thupi great job with the thorough testing! I just have a quick question about one of the describe blocks. After that we should be good to go :)

**Comment by thupi** [comment]
@mark-drastrup Good catch 😃 Seems like something went wrong as I was copying around templates for the tests. I tried to correct so that it correctly tests the different types of inputs as with the size tests 😄

**Comment by mark-drastrup** [comment]
@thupi ahh I see 😄 It looks great now! I think we're ready to go 😄

**Review by mark-drastrup — APPROVED** [review]

**Pull request merged**